src/test/java/io/cdap/delta/bigquery/BigQueryConsumerTest.java

```diff
@@ -126,6 +128,7 @@ public void setup() throws Exception {
     Mockito.when(dataFileWriter.create(Mockito.any(org.apache.avro.Schema.class), Mockito.any(OutputStream.class)))
       .thenReturn(dataFileWriter);
     PowerMockito.whenNew(DataFileWriter.class).withAnyArguments().thenReturn(dataFileWriter);
+    PowerMockito.whenNew(MultiGCSWriter.class).withAnyArguments().thenReturn(gcsWriter);
```

Comment:
Instead of this can we just mock `dataFileWriter.append` method to avoid using powermock for MultiGCSWriter?

Reply:
I didn't get this. gcsWriter.write operation is performed which has to be mocked.How can it be replaced by this.

Reply:
It can be replaced because if you trace the call chain eventually it will reach to `dataFileWriter.append`